src/test/java/com/razorpay/StakeholderClientTest.java

```diff
@@ -200,4 +200,53 @@ private String getResponseCollection(){
                 "  \"count\": 1\n" +
                 "}";
     }
+
+    @Test
+    public void testuploadStakeholderDoc() throws RazorpayException {
+        JSONObject request = new JSONObject();
+        request.put("files","/Users/your_name/Downloads/sample_uploaded.pdf");
+        request.put("document_type","aadhar_front");
+
+        String mockedResponseJson = "{\n" +
+                "  \"entity\": \"account\",\n" +
+                "  \"individual_proof_of_address\": [\n" +
+                "    {\n" +
+                "      \"type\": \"aadhar_front\",\n" +
+                "      \"url\": \"https://rzp.io/i/bzDAbNg\"\n" +
+                "    }\n" +
+                "  ]\n" +
+                "}";
```

Comment:
please check if mockedResponseJson can be created as new JSONObject, then convert it to String using toString() method so that it can be passed to mockResponseFromExternalClient()

Reply:
changed json string to json object

Reply:
https://github.com/razorpay/razorpay-java/blob/530334834fab1783cb3abd7e26e87a56681d2383/src/test/java/com/razorpay/StakeholderClientTest.java#L218